### ISSUE TITLE
TypeError: this$1.innerListener_.unlisten is not a function

### ISSUE THREAD

**Issue by wussup** [mapgears/ol3-google-maps#280]
Hello, I have an issue when try to set a new view on the map:

> vue.runtime.esm.js?2b0e:1888 TypeError: this$1.innerListener_.unlisten is not a function
>     at Map.eval (PropertyListener.js?4e70:12)
>     at Map.boundListener (events.js?1e8d:41)
>     at Map.dispatchEvent (Target.js?0ec0:99)
>     at Map.notify (Object.js?e269:151)
>     at Map.set (Object.js?e269:170)
>     at Map.setView (PluggableMap.js?fe37:1300)

PropertyListener has next code:

```
Listener.call(this, target.on('change:' + key, function (e) {
      if (this$1.innerListener_) {
        this$1.innerListener_.unlisten();
      }
      this$1.innerListener_ = listen(e.target.get(e.key), e.oldValue);
    }));
```

The problem is here:

`this$1.innerListener_.unlisten();`

Probably because this$1.innerListener_ is a table of listeners:

> 0: PropertyListener {listenerKey_: {…}, innerListener_: undefined}
> 1: PropertyListener {listenerKey_: {…}, innerListener_: undefined}
> length: 2

I do it to change the projection of the map.

Versions:
- olgm -> 1.5.0
- ol -> 5.3.3

**Comment by adube** [comment]
An extra "if" could be added there to check if the innerListener_ is an array and loop if it is.  Would you please work on a fix?  I'd gladly review it.

**Comment by wussup** [comment]
Ready, https://github.com/mapgears/ol3-google-maps/pull/281

**Comment by adube** [comment]
Fixed by #281
